indexing-service/src/main/java/io/druid/indexing/overlord/http/OverlordResource.java

```diff
@@ -373,6 +370,32 @@ public Response getRunningTasks()
   }
 
   @GET
+  @Path("/incompleteTasks")
+  @Produces(MediaType.APPLICATION_JSON)
+  public Response getIncompleteTasks()
+  {
+    final List<TaskResponseObject> incompleteTasks = Lists.transform(
+        taskStorageQueryAdapter.getActiveTasks(),
```

Comment:
Nothing in the call train seems to stop exceptions. Are you intending to have the last-resort exception reporter handle any exceptions?